spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :feature do
+  before do
+    @user1 = User.create(name: 'Tom', photo: 'https://via.placeholder.com/150', bio: 'I love coding!',
+                         posts_counter: 2)
+    @user2 = User.create(name: 'Sana', photo: 'https://via.placeholder.com/150', bio: 'I love reading!',
+                         posts_counter: 0)
+  end
+
+  describe 'User index page' do
+    it 'displays the username of all other users' do
+      visit users_path
```

Comment:
- [ ] Kindly implement the missing tests highlighted below as stated in the requirement. 🙏 

<img width="672" alt="Untitled" src="https://github.com/iankiema/BlogApp/assets/103386183/225376ed-8399-46ce-9068-7474a87ebecc">